lms/models/lti_params.py

```diff
@@ -77,6 +85,8 @@ def _to_lti_v11(v13_params):
             if path_item not in value:
                 break
             value = value[path_item]
+        else:
```

Comment:
Added a test for this. 

We only want to translate a key if we found the whole path. The code was trying to do that but always taking the first match anyway.